go.mod

```diff
@@ -10,10 +10,10 @@ require (
 	github.com/hashicorp/go-multierror v1.1.1
 	github.com/kserve/kserve v0.15.0
 	github.com/kuadrant/authorino v0.20.0
-	github.com/kubeflow/model-registry v0.2.14
+	github.com/kubeflow/model-registry v0.2.7-alpha.0.20250521083438-bdcbb032a58e
```

Comment:
It is confusing, why do we have an older version here?
I think we had this discussion one time :D

Reply:
yeah we did 😁

on model registry we have release tags on branches now (e.g. `v0.2.18` is tagged on branch `release/v0.2.18`)

so when using go get model-registry@commit-hash , and this commit is on the main branch 
go takes the latest tag on the main branch that in our case is 0.2.7-alpha 🤯 

if you want to see a newer tag here, we should wait for the next upstream release of model registry

Reply:
what about using v0.0.0- like other deps in go.mod pointing to commit hashes ? @spolti

Reply:
Can it be fixed by creating a new tag from the main branch?

> if you want to see a newer tag here, we should wait for the next upstream release of the model registry

I don't want to block you guys, but having the correct version would be nice.
I'll delegate the decision to merge it to your guys :)

Reply:
i got confused yesterday with the same reason. 
i guess with a comments for the reason can be helpful and update here once again when next release is out.

Reply:
yes fine by me, I'll put a comment above that line to explain why and then update it after the next release if it's ok to you folks

Reply:
done